packages/alfa-rules/src/sia-r70/rule.ts

```diff
@@ -0,0 +1,141 @@
+import { Rule, Diagnostic } from "@siteimprove/alfa-act";
+import { Document, Element, Namespace } from "@siteimprove/alfa-dom";
+import { Predicate } from "@siteimprove/alfa-predicate";
+import { Err, Ok } from "@siteimprove/alfa-result";
+import { Page } from "@siteimprove/alfa-web";
+import { List } from "@siteimprove/alfa-list";
+import { Iterable } from "@siteimprove/alfa-iterable";
+
+import { expectation } from "../common/expectation";
+import { hasChild, isRendered, isDocumentElement } from "../common/predicate";
+import { Group } from "../common/group";
+
+const { isElement, hasName, hasNamespace } = Element;
+const { and, test } = Predicate;
+
+const deprecated = [
+  "applet",
+  "basefont",
+  "bgsound",
+  "big",
+  "blink",
+  "center",
+  "content",
+  "dir",
+  "font",
+  "frame",
+  "frameset",
+  "hgroup",
+  "image",
+  "keygen",
+  "marquee",
+  "menuitem",
+  "nobr",
+  "noembed",
+  "noframes",
+  "plaintext",
+  "rb",
+  "rtc",
+  "shadow",
+  "spacer",
+  "strike",
+  "tt",
+  "xmp",
+];
+
+export default Rule.Atomic.of<Page, Document>({
+  uri: "https://alfa.siteimprove.com/rules/sia-r70",
+  evaluate({ device, document }) {
+    return {
+      applicability() {
+        return test(hasChild(isDocumentElement), document) ? [document] : [];
+      },
+
+      expectations(target) {
+        const deprecatedElements = target
+          .descendants({ flattened: true, nested: true })
+          .filter(isElement)
+          .filter(
+            and(
+              hasNamespace(Namespace.HTML),
+              hasName("acronym", ...deprecated),
+              isRendered(device)
+            )
+          );
+
+        return {
+          1: expectation(
+            deprecatedElements.isEmpty(),
+            () => Outcomes.IsNotDeprecated,
+            () => Outcomes.IsDeprecated(deprecatedElements)
+          ),
+        };
+      },
+    };
+  },
+});
+
+export namespace Outcomes {
+  export const IsNotDeprecated = Ok.of(
```

Comment:
Let's change the name to `HasNoDeprecatedElement`.
It's not about the target (the page) being or not deprecated, but having or not deprecated elements.